### PR TITLE
Feature comments

### PR DESCRIPTION
**Pull request by Charchit507**

### CODE REVIEW

File: app/views/comments/_comment.html.erb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <p>
   <strong>Commenter:</strong>
-  <%= comment.commenter %>
+  <%= comment.user.first_name %>&nbsp<%= comment.user.last_name %>
```
Comment (neeteshgupta95) [inline app/views/comments/_comment.html.erb]: create one method in the User model with `full_name` and use same method here

**Review by neeteshgupta95 — CHANGES_REQUESTED** [review]

File: app/models/user.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  def fullname
+    self.first_name << " " << self.last_name
```
Comment (neeteshgupta95) [inline app/models/user.rb]: 1. remove self 
2 use interpolation

File: app/models/user.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  def fullname
```
Comment (neeteshgupta95) [inline app/models/user.rb]: rename it to `full_name`

**Review by neeteshgupta95 — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 2 above)
Comment (Charchit507) [reply app/models/user.rb]: okay

File: app/models/user.rb (hunk 3 above)
Comment (Charchit507) [reply app/models/user.rb]: okay

**Review by neeteshgupta95 — APPROVED** [review]

**Pull request merged**